settings.py.template

```diff
@@ -1,11 +1,4 @@
 import os
```

Comment:
We should probably remove this file entirely, if we're switching to secrets manager.

Reply:
ah yeah, done!